vm/ubpf_jit_x86_64.c

```diff
@@ -867,6 +867,11 @@ ubpf_translate_x86_64(struct ubpf_vm* vm, uint8_t* buffer, size_t* size, char**
     state.jumps = calloc(UBPF_MAX_INSTS, sizeof(state.jumps[0]));
     state.num_jumps = 0;
 
+    if (!state.pc_locs || !state.jumps) {
```

Comment:
Same bug exists in ubpf_jit_arm64.c. Can you fix that as well?

Reply:
Done